.github/workflows/ci.yml

```diff
@@ -303,15 +311,11 @@ jobs:
           New-Item -Path '${{ matrix.unity-config-path }}' -ItemType Directory
           Set-Content -Path '${{ matrix.unity-config-path }}services-config.json' -Value '${{ secrets.UNITY_LICENSE_SERVER_CONFIG }}'
 
-      # Step used to reduce the Unity path to avoid long path errors on Windows.
-      - name: Make symbolic link for Unity (Windows)
-        run: New-Item -ItemType SymbolicLink -Path "C:\${{ steps.env.outputs.unityVersion }}" -Target "C:\Program Files\Unity\Hub\Editor\${{ steps.env.outputs.unityVersion }}"
```

Comment:
1. it was a temporary fix AFAICT - https://github.com/getsentry/sentry-unity/pull/423
2. it wasn't even used anymore - originally it was in the `build` job and was supposed to stay there - here, it was completely ignored because we passed the full unity path anyway.